workflows/scripts/consensus.py

```diff
@@ -31,7 +31,7 @@ def build_consensus(
     """
 
     # initiating Logger
-    log_path = Path(log_file).absolute()
+    log_path = pathlib.Path(log_file).absolute()
```

Comment:
Seeing this logging block repeat, is there a way to define logging configuration in one spot and reference it from multiple locations so as to decrease maintenance and increase uniform treatment of logging?

Reply:
That's correct. I have taken note of this in the #14